workflow-templates/im-deploy-az-app-service-manually-to-prod-subn.yml

```diff
@@ -140,13 +138,23 @@ jobs:
         with:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
 
+      # TODO:  If there is only one environment in the prod subscription, the following steps can be removed and a single
+      # environment variable can be added at the workflow level above for RESOURCE_GROUP, AZ_FUNC_NAME
       - uses: im-open/set-variable-based-on-environment@v1.0.0
         with:
           variable-name: 'RESOURCE_GROUP'
           current-environment: ${{github.event.inputs.environment}}
           stage-value: '' # TODO: Add the stage rgrp name or delete if env does not exist
           prod-value: '' # TODO: Add the prod rgrp name
 
+      - name: Set the Azure Function Name
+        uses: im-open/set-variable-based-on-environment@v1.0.0
+        with:
+          variable-name: 'AZ_APP_NAME'
```

Comment:
The comment says `AZ_FUNC_NAME` but the action says `AZ_APP_NAME`. Is one of them wrong?

Reply:
I swear I checked the four files very carefully.  I guess I didn't check good enough.  I'll fix this.